src/elements/select/select-button.svelte

```diff
@@ -0,0 +1,32 @@
+
+<svelte:options immutable tag='v-select-button' />
+
+<script lang='ts'>
+
+import { addStyles } from '../../lib/index';
+import { dispatcher } from '../../lib/dispatch';
+
+
+export let buttontext = 'ENTER';
+export let buttonicon = '';
+
+const dispatch = dispatcher();
+
+addStyles();
+
+const handleButtonClick = () => {
+  dispatch('button-click');
```

Comment:
We don't need this dispatcher or `button-click` event, since the consumer of the component can just do:
```
<v-select-button on:click={handleClick} />
```

Reply:
Do you know why it works when I use `v-select-button` but it doesn't when I import it as a `SelectButton`?